### ISSUE TITLE
No module named angr.storage.memory

### ISSUE THREAD

**Issue by bannsec** [andreafioraldi/angrdbg#6]
Looks like angr's APIs changed on you.

```
  File "/home/angr/.virtualenvs/angr/lib/python3.8/site-packages/angrdbg/memory_8.py", line 17, in <module>
    from angr.storage.memory import SimMemory, DUMMY_SYMBOLIC_READ_VALUE
ModuleNotFoundError: No module named 'angr.storage.memory'
```

**Comment by andreafioraldi** [comment]
Yes, it works with angr 8 but not 9 I guess, I don't have time or desire to update this repo. If you need it, feel free to open a PR, I still accept contributions gladly.
Otherwise I should insert the version of angr that is required in setup.py, but I don't know the last angr 8 release number as they don't use the github release page :(

**Comment by oToToT** [comment]
@andreafioraldi From the [release page](https://pypi.org/project/angr/#history) in PyPI, I guess that it should be 8.20.7.27.

**Comment by jlevis73** [comment]
Has anyone looked into how difficult it would be to update for angr v9 compatibility? Trying to evaluate if it's worth doing myself or just downgrading to angr v8.

**Comment by bannsec** [comment]
I basically switched over to symbion, but that is having issues lately too.
Not sure there's a good working solution anymore.
